Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -56,7 +56,7 @@ moduleConfig :
 ;
 
 moduleConfigProperty :
-    BEGINPROPERTY whiteSpace unrestrictedIdentifier (whiteSpace GUIDLITERAL)? endOfStatement
+    BEGINPROPERTY whiteSpace unrestrictedIdentifier (LPAREN numberLiteral RPAREN)? (whiteSpace GUIDLITERAL)? endOfStatement
```

Comment:
Just making sure --- is it legal to have whitespace between the identifier and the `(`? It might get pretty-printed out but the grammar tries to work even with non-pretty-printed version.

Reply:
Whitespace causes VB6 to warn on open and emit to a log:

    Line 43: The CLSID (1) for ColumnHeader is invalid.
    Line 44: Syntax error: property {0713E8C7-850A-101B-AFC0-4210102A8DA7} in ColumnHeader was missing an '='.

It repairs it on save. Should I allow in the grammar?

Reply:
I wouldn't. From the warning message, the grammar has changed, so it's fine as it is.